### PR TITLE
Updated language.c for Traditional Chinese

### PR DESCRIPTION
**Pull request by tommai78101**

### CODE REVIEW

**Comment by tommai78101** [comment]
Let me know if there's more strings needed to be added for Traditional Chinese.

**Comment by yellows8** [comment]
This should be in #19.

**Comment by tommai78101** [comment]
@yellows8 I tried, but I don't exactly know how to merge #33 into #19.

File: common/language.c [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿#include "language.h"
+    ﻿#include "language.h"
```
Comment (yellows8) [inline common/language.c]: Remove whitespace.

**Comment by tommai78101** [comment]
DONE! Anything else?

**Comment by yellows8** [comment]
Regarding merging-PRs: You can do that via PRing this to here: https://github.com/AntonioDePau/nx-hbmenu/tree/patch-2

**Comment by yellows8** [comment]
Just merging this since {PR to above repo} had zero response.

**Pull request merged**